google/resource_dataflow_job.go

```diff
@@ -0,0 +1,210 @@
+package google
```

Comment:
You'll also want to add documentation in website/docs/r/

Reply:
I added this last night but forgot to comment.